etc/utils.py

```diff
@@ -0,0 +1,48 @@
+from etc.consts import WORDS_FILE_NAME, VALID_WORD_RX
+from enum import Enum
+import random
+import re
+
+
+class Color(Enum):
+    GREY = 0
+    YELLOW = 1
+    GREEN = 2
+
+
+def get_random_word(words):
+    """
+    Given a list of words, returns a random word.
+
+    :param words: list of words (strings)
+    :return: a random word from words list
+    """
+    if not words:
+        raise Exception("Words list is empty!", words)
+
+    rand_idx = random.randint(0, len(words)-1)
```

Comment:
Look into `random.choice`

Reply:
Done😃
random word is being called once per game, I guess it won't affect the program's efficiency too much?

Reply:
No, it's just a matter of readability.

Reply:
+1